dockerfiles/compilation/java-8/compile.sh

```diff
@@ -77,4 +78,10 @@ if [ $? -eq 0 ]; then
       echo "Copying SPRING jars"
       cp server/app/target/james-server-app-*-app.zip $SPRING_DESTINATION
    fi
+
+   if [ -d "$SWAGGER_DESTINATION" ]; then
+      cp server/protocols/webadmin/webadmin-data/target/webadmin-data.json $SWAGGER_DESTINATION || true
+      cp server/protocols/webadmin/webadmin-mailbox/target/webadmin-mailbox.json $SWAGGER_DESTINATION || true
+      cp server/protocols/webadmin/webadmin-swagger/target/webadmin-swagger.json $SWAGGER_DESTINATION || true
```

Comment:
in SwaggerRoutes it scanned all routes before generate json file so I think we will have only one json file with different tag name. WDYT?

Reply:
This tools is just scanning the source code for swagger annotations.
I wanted a simple tools such this one, as I don't want to do some code execution, manager injection...

The benefit of this tool is also to separate the documentation by projects, then you will know which project you have to install while building your product; in order to match your needs.